src/util/selinux.c

```diff
@@ -340,6 +341,15 @@ static int bus_selinux_log(int type, const char *fmt, ...) {
         return 0;
 }
 
+/**
+ * On a policy reload we need to reparse the SELinux configuration file, since
+ * this could have changed. The call back is registered in the broker, but
+ * we need to reload the configuration in the launcher.
+ */
+static int policy_reload_callback(int seqno) {
+	return raise(SIGHUP);
```

Comment:
What does SELinux do with return values from this callback?

Reply:
In our case nothing.  If the callback returns non-zero `selinux_status_updated` returns -1, but the dbus-broker code isn't calling `selinux_status_updated` directly.  Its calling it through `selinux_check_access` which ignores the return.